src/markdown/downloads/federal-production-by-month.md

```diff
@@ -47,19 +47,19 @@ The sum of 12-month production totals for a given calendar or fiscal year may no
 
 ### Fields and definitions
 
-_Month_ The month for which the production was reported.
+_Month_ The month for which the production occurred.
 
-_Calendar Year_ The calendar year for the month in which the production was reported.
+_Calendar Year_ The calendar year for the month in which the production occurred.
 
 _Land Class_ This field distinguishes federal lands and waters from Native American lands.
 
-_Land Category_ This field distinguishes between onshore and offshore revenue. Some revenues – such as civil penalties – are not tied to either onshore or offshore.
+_Land Category_ This field distinguishes between onshore and offshore production.
 
 _Commodity_ We have monthly data for oil, gas, and coal, since these are high-volume commodities that result in the most revenue. This field includes the units for each commodity in parentheses.
 
-- _Oil Prod Vol (bbl)_ Oil production is measured in barrels (bbl)
-- _Gas Prod Vol (Mcf)_ Gas (natural gas) is measured in thousand cubic feet (mcf)
-- _Coal Prod Vol (ton)_ Coal is measured in tons. A ton is 2000 pounds.
+  - _Oil Prod Vol (bbl)_ Oil production is measured in barrels (bbl)
+  - _Gas Prod Vol (Mcf)_ Gas (natural gas) is measured in thousand cubic feet (mcf)
```

Comment:
@lpgoldstein It looks like `mcf` is lowercase in the date file (unless I'm looking at the wrong file). We should probably case it the same here as in the data file.